treeherder_config.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+TREEHERDER_CONFIG = os.environ.get('TREEHERDER_CONFIG') or 'credentials.ignore'
+
+config = {
+    #"treeherder_url": "http://local.treeherder.mozilla.org",
+    "treeherder_url": "https://treeherder.allizom.org",
+
```

Comment:
How about keep a default setting in this config file and provide a command-line option for `treeherder_url`?